tests/integration/tests/accounts-to-ciphers.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../boot'
+require 'minitest/autorun'
+require 'pry-rescue/minitest' unless ENV['CI']
+require 'pry'
+require 'uri'
+
+def setup_ciphers(override_account_attrs = {})
+  inst = Instance.create name: "Alice"
+
+  bw = Bitwarden.new inst
+  bw.login
+  assert_equal bw.sync, "Syncing complete."
+
+  source_url = "file://" + File.expand_path("../konnector", __dir__)
+  inst.install_konnector "bankone", source_url
+
+  account_attrs = {
+    :type => "bankone",
+    :name => "Bank one",
+    :auth => {:login => "Isabelle", :zipcode => "64000"}
+  }
```

Comment:
[nit] In ruby, there is an alternative syntax for hashs when the keys are symbols:

```ruby
  account_attrs = {
    type: "bankone",
    name: "Bank one",
    auth: { login: "Isabelle", zipcode: "64000" }
  }
```

Reply:
Oh, much simpler, thanks.